src/cli/bin.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict'
 
+require('pipe-args').load()
```

Comment:
there are commands like `ipfs swarm peers` that should not take input from stdin